regression_test.go

```diff
@@ -0,0 +1,79 @@
+package veneur
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	proto "github.com/golang/protobuf/proto"
+	"github.com/stripe/veneur/samplers"
+	"github.com/stripe/veneur/ssf"
+)
+
+func validSample() ssf.SSFSpan {
+	return *&ssf.SSFSpan{
+		TraceId:        1,
+		Id:             1,
+		StartTimestamp: 1,
+		EndTimestamp:   10,
+	}
+}
+
+func TestTagNameSet(t *testing.T) {
+	sample := validSample()
+	sample.Tags = make(map[string]string)
+	sample.Tags["name"] = "testName"
+
+	t.Run("nameNotSet", func(t *testing.T) {
```

Comment:
Could you add a comment explaining what this is meant to test? (I think this one is testing that a tag becomes the name, but leaves the tag?)